dart_flexible_sync/README.md

```diff
@@ -69,12 +69,6 @@ Then run this command:
     * Press the "Flexible Sync" button. Only Flexible Sync is supported in the Realm Dart SDK.
     * Switch ON the ["Development mode"](https://www.mongodb.com/docs/atlas/app-services/sync/data-model/development-mode/) option.
     * Create a new database collection and choose a name for it.
-    * Create a new queryable field used for filtering data between both realms in this application. For this sample the fields are `status` and `progressMinutes`.
-        Type the field name `status` in the selection box and then choose `Create status`.
-        It will be created.
```

Comment:
I'm wondering if we should talk a bit about the implications of turning on development mode, ie. that:

1. Queryable fields are inferred
2. Additive Schema changes accepted

Reply:
Ok